services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -806,67 +806,20 @@ private void createExecutionEngineChannel() {
     Optional<SpecLogic> specLogic = spec.getSpecLogicForMilestone(SpecMilestone.MERGE);
     if (specLogic.isPresent() && beaconConfig.powchainConfig().isEnabled()) {
       executionEngineChannel =
-          ExecutionEngineService.create(beaconConfig.powchainConfig().getEth1Endpoints().get(0));
+          ExecutionEngineChannelImpl.create(
+              beaconConfig.powchainConfig().getEth1Endpoints().get(0));
```

Comment:
there will be maybe a future config dedicated to the execution engine endpoint?

Reply:
Yep, I think this is more like a stub option now (just copied it from original version). 
@mkalinin do you think it's worth leaving this 'as is' for interop ?